plugins/instructions/SOS/README.md

```
@@ -0,0 +1,21 @@
+# Setting Up SOS
+
+1. Follow authentication steps
+2. When in situations of dangers, say "SOS"
+3. When the memory is created, it will notice the SOS, and contact your emergency contact
+
+## Authentication 
+
+
+1. Open (https://sos-auth.vercel.app/)[https://sos-auth.vercel.app/]
```

Markdownlint

(https://sos-auth.vercel.app/)[https://sos-auth.vercel.app/]
Reversed link syntax

(MD011, no-reversed-links)